data/vectorojay.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Vector Ojay",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Photographer, transitioning to Tech || Software Developer(TypeScript, React). Versatile. Inquisitive. Passionate about Designs and UI. Eager for knowledge."
+  "avatar": "https://github.com/vectorojay.png"
+  "links" : [
+  {
+  "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/vectorojay/",
+      "icon": "FaLinkedinIn"
```

Comment:
And also change the value of icon it should be **FaLinkedin**! thank you for contributing.